ThatDefaultProject/Assets/ThatDefaultProject/Scripts/TransformMimic.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace that
+{
+    public class TransformMimic : MonoBehaviour
+    {
+        [Header("Mimic")]
+        [SerializeField] private Transform _transformToMimic;
+        [Header("Settings")]
+        [SerializeField] private bool _mimicPosition = true;
+        [SerializeField] private bool _mimicRotation = true;
+
+        void Update()
+        {
+            if (_transformToMimic == null) return;
+
+            if (_mimicPosition)
+            {
+                transform.position = _transformToMimic.position;
```

Comment:
Maybe add an offset as well so you can easily have e.g. a camera follow a player or object using a set offset

Reply:
That is indeed a good suggestion